### PR TITLE
Feature: Display rules

### PR DESCRIPTION
**Pull request by mayank2913**
- [ ] Apply Rules API Added
- [ ] Rules Integrated into the frontend
- [ ] Filter Implemented
- [ ] Aligned the no-issues Image & Icon to center
- [ ] Test Cases for applyRule API added
- [ ] View Rule Added
- [ ] Delete Rule API, Test Cases, and UI Integration
- [ ] Object Detail Toggle

### CODE REVIEW

File: ui/src/app/components/add-index-form/add-index-form.component.ts [hunk 1]
```diff
@@ -39,6 +45,33 @@ export class AddIndexFormComponent implements OnInit {
         if (res !== '') this.selectedTableChange(res)
       },
     })
+
```
Comment (shreyakhajanchi) [inline ui/src/app/components/add-index-form/add-index-form.component.ts:48]: Clicking on add index after deleting a rule is again opening delete rule side nav

File: webv2/utilities/utilities.go [hunk 2]
```diff
@@ -299,6 +299,7 @@ func GetFilePrefix(now time.Time) (string, error) {
 	return dbName + ".", nil
 }
 
+//Remove after implementing rules display
```
Comment (shreyakhajanchi) [inline webv2/utilities/utilities.go:302]: can this method be removed now ?

File: ui/src/app/services/data/data.service.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { Injectable } from '@angular/core'
```
Comment (shreyakhajanchi) [inline ui/src/app/services/data/data.service.ts:1]: Please update dist/ files before merging the PR

File: webv2/utilities/utilities.go (hunk 2 above)
Comment (mayank2913) [reply webv2/utilities/utilities.go]: Updated

File: ui/src/app/services/data/data.service.ts (hunk 3 above)
Comment (mayank2913) [reply ui/src/app/services/data/data.service.ts:1]: Will handle before the Merge.

File: ui/src/app/components/object-detail/object-detail.component.scss [hunk 4]
```diff
@@ -273,3 +273,15 @@ p {
 .scr-column-data-edit-mode {
   height: 58.3px;
 }
+
+#middle-column-toggle-button {
+  z-index: 100;
+  
+  // right: 4px;
```
Comment (shreyakhajanchi) [inline ui/src/app/components/object-detail/object-detail.component.scss:280]: remove the commented code

File: ui/src/app/components/object-detail/object-detail.component.scss [hunk 5]
```diff
@@ -273,3 +273,15 @@ p {
 .scr-column-data-edit-mode {
   height: 58.3px;
 }
+
+#middle-column-toggle-button {
```
Comment (shreyakhajanchi) [inline ui/src/app/components/object-detail/object-detail.component.scss:277]: Please move these css to styles.css

File: ui/src/app/components/object-detail/object-detail.component.ts [hunk 6]
```diff
@@ -1168,6 +1169,12 @@ export class ObjectDetailComponent implements OnInit {
     this.currentTabIndex = tabChangeEvent.index
   }
 
+  middleColumnToggle() {
+    console.log('hi')
```
Comment (shreyakhajanchi) [inline ui/src/app/components/object-detail/object-detail.component.ts:1173]: please remove the console logs

File: ui/src/app/components/workspace/workspace.component.scss [hunk 7]
```diff
@@ -66,6 +66,14 @@ app-object-detail {
   width: 97%;
 }
 
+.middle-column-hide{
+  width: 0%;
+}
+
+.middle-column-full{
+  width: 100%;
+}
```
Comment (shreyakhajanchi) [inline ui/src/app/components/workspace/workspace.component.scss:75]: Please move these CSS to styles.css

File: ui/src/app/components/workspace/workspace.component.ts [hunk 8]
```diff
@@ -202,11 +208,20 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
...
+    setTimeout(() => {
+      this.rulesLabel = `RULES (${count})`
+    })
+  }
   leftColumnToggle() {
     this.isLeftColumnCollapse = !this.isLeftColumnCollapse
   }
 
+  middleColumnToggle() {
+    console.log('changed')
```
Comment (shreyakhajanchi) [inline ui/src/app/components/workspace/workspace.component.ts:221]: remove this console log too

**Review by shreyakhajanchi — COMMENTED** [review]
Largely LGTM, left minor comments. Also update the dist files that are getting generated, getting different ones of my system then the one in PR

File: ui/src/app/components/workspace/workspace.component.ts (hunk 8 above)
Comment (mayank2913) [reply ui/src/app/components/workspace/workspace.component.ts]: updated

File: ui/src/app/components/object-detail/object-detail.component.ts (hunk 6 above)
Comment (mayank2913) [reply ui/src/app/components/object-detail/object-detail.component.ts]: updated

File: ui/src/app/components/object-detail/object-detail.component.scss (hunk 4 above)
Comment (mayank2913) [reply ui/src/app/components/object-detail/object-detail.component.scss]: updated

File: ui/src/app/components/workspace/workspace.component.scss (hunk 7 above)
Comment (Ronak-Searce) [reply ui/src/app/components/workspace/workspace.component.scss:75]: Resolved

File: ui/src/app/components/object-detail/object-detail.component.scss (hunk 5 above)
Comment (Ronak-Searce) [reply ui/src/app/components/object-detail/object-detail.component.scss]: resolved

**Review by shreyakhajanchi — APPROVED** [review]
LGTM

**Pull request merged**